frontend/app/forms/inputs/checkbox.component.js

```diff
@@ -4,29 +4,24 @@ import { get } from "lodash";
 import { DataContainerContext } from "../data-container.component";
 
 export default function Checkbox(props) {
-  const [value, setValue] = React.useState(
-    get(props.data, props.dataKey, false)
```

Comment:
previously it never had the right initial value, since it was looking in props instead of context

Reply:
👍 good catch -- we recently switched from props to context and must have missed this.